src/Workspaces/Core/Portable/Execution/AbstractReferenceSerializationService.cs

```diff
@@ -101,13 +101,17 @@ public Checksum CreateChecksum(AnalyzerReference reference, CancellationToken ca
                         WriteUnresolvedAnalyzerReferenceTo(unresolved, writer);
                         break;
 
+                    case AnalyzerReference analyzerReference when analyzerReference.GetType().FullName == "Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem.VisualStudioUnresolvedAnalyzerReference":
```

Comment:
:bulb: Move to constant?

❔ Where is this type defined? Is there no way we can make it implement `UnresolvedAnalyzerReference` or otherwise reduce the likelihood of regression?